pkg/cmd/admin/router/router.go

```diff
@@ -250,6 +257,8 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out, errout io.
 		StatsPort:     defaultStatsPort,
 		HostNetwork:   true,
 		HostPorts:     true,
+
+		MaxConnections: defaultMaxConnections,
```

Comment:
Basically, client code that has server side components should never impose defaults - the server should do that.  Same as with validation.  Client side API logic is not appropriate.